packages/react/src/components/ComboBox/ComboBox.test.tsx

```diff
@@ -530,6 +530,24 @@ describe('ComboBox', () => {
     expect(combobox.getAttribute('value')).toEqual('One');
   });
 
+  it('Updates aria-activedescendant with keyboard', () => {
+    const options: IComboBoxOption[] = [
+      { key: '1', text: '1', id: 'one' },
+      { key: '2', text: '2', id: 'two' },
+      { key: '3', text: '3', id: 'three' },
+    ];
+    const { getByRole } = render(<ComboBox defaultSelectedKey="1" options={options} />);
+    const combobox = getByRole('combobox');
+    // open combobox
+    userEvent.tab();
+    userEvent.keyboard('{enter}');
+    expect(combobox.getAttribute('aria-expanded')).toEqual('true');
```

Comment:
Should we add a check that `aria-activedescendant` is equal to `one` here in case something changes in regards to the ids of the test's options in the future? (Unlikely, but I just want to make sure we account for all of these cases)

```suggestion
    expect(combobox.getAttribute('aria-expanded')).toEqual('true');
```
```suggestion
    expect(combobox.getAttribute('aria-expanded')).toEqual('true');
    expect(combobox.getAttribute('aria-activedescendant')).toEqual('one');
```